README.md

```diff
@@ -61,55 +65,71 @@ To remove all containers run:
 
     $ docker-compose rm
 
-It is also possible to use Docker Swarm instead of Docker Compose for bigger
-setups. Let us know if this is interesting for you.
+To remove the database you have to remove the content of the `db-data`
+directory.
 
 
-## Configuration of the generated YAML file
+## Configuration of the generated Docker Compose YAML file
 
-The setup command generates a YAML file (default filename: `docker-compose.yml`)
-with the container configuration for all services. This step can be configured
-with a YAML formated config file. E. g. to get a customized YAML file run:
+The `setup` command generates a Docker Compose YAML file (default filename:
+`docker-compose.yml`) with the container configuration for all services. This
+step can be configured with a (second) YAML formated configuration file.
+
+E. g. create a file `my-config.yml` like this:
+
+    ---
+    port: 9000
+
+See the [default config](pkg/config/default-config.yml) for syntax and defaults
+of this file.
```

Comment:
I am not absolutely against this. But please think about this problem: if you checkout an older version (or the master has changed) then the configuration variables could also have changed. So the link could be on an other version then the version the user is using.

One solution would be to document the file in this readme. Another solution would be to document it in the manage tool. For example with

openslides config --help

Then you could write here, that the user should type this command to get the documentation.